draft-irtf-cfrg-hpke.md

```diff
@@ -1574,10 +1558,20 @@ Initial contents: Provided in {{aead-ids}}
 # Acknowledgements
 
 The authors would like to thank
+Joel Alwen,
+Jean-Philippe Aumasson,
 David Benjamin,
 Benjamin Beurdouche,
 Frank Denis,
```

Comment:
I'd like to add
```
Bruno Blanchet,
Eduard Hauck,
Doreen Riepel,
```
Could you please add them to this PR? :) (at the right spot according to alphabetic order)

Reply:
Done!